tests/test_quick_run_config_generator.py

```diff
@@ -911,6 +930,138 @@ def _get_next_run_config_ensemble(self,
 
         self.assertEqual(perf_config['batch-size'], 1)
 
+    def _get_next_run_config_bls(self, max_concurrency=0, min_concurrency=0):
+        """
+        Test that get_next_run_config() creates a proper RunConfig for BLS
+
+        Sets up a case where the coordinate is [1,2,3,4,5,6], which corresponds to
+          - model max_batch_size = 2
+          - model instance count = 3
+          - model concurrency = 2*3*2 = 12
+          - composing model 1 max_batch_size = 8
+          - composing model 1 instance_count = 5
+          - composing model 1 concurrency = 8*5*2 = 80
+          - composing model 2 max_batch_size = 32
+          - composing model 2 instance_count = 7
+          - composing model 2 concurrency = 32*7*2 = 448
+          - model concurrency = 12 (minimum value of [12, 80, 448])
+
+        Also,
+        - sequence batching should be on for model 1
+        - dynamic batching should be on for model 2
+        - existing values from the base model config should persist if they aren't overwritten
```

Comment:
Are these two statements tested below?

Reply:
Yes, these are the lines that check it:
``` 
self.assertEqual(composing_model_config0.to_dict(),
                         expected_model_config0)
self.assertEqual(composing_model_config1.to_dict(),
                         expected_model_config1)
```

Reply:
Good catch. Base model config check was missing. I've added it. Relevant PA config settings are checked (concurrency/batch size) here:
```
if max_concurrency:
            self.assertEqual(perf_config['concurrency-range'], max_concurrency)
        elif min_concurrency:
            self.assertEqual(perf_config['concurrency-range'], min_concurrency)
        else:
            self.assertEqual(perf_config['concurrency-range'], 12)

        self.assertEqual(perf_config['batch-size'], 1)
```